client/src/daemon_launcher.cpp

```diff
@@ -38,7 +44,51 @@ pid_t daemon_launcher::start() {
     return ret;
 }
 
-#ifndef __APPLE__
+#include <iostream>
+
+#ifdef __APPLE__
```

Comment:
In my opinion, this #ifdef should be moved to the method body, instead of having it outside the function.